### DISCUSSION TITLE
[Projects Beta] Settings Button doesn't work

### DISCUSSION THREAD

**Discussion by maxcell** [community/community#6834] [Projects and Issues] [answered]
When I click on the Settings Button for a project, nothing happens. No network requests or anything.

This happens on:
- Chrome Version 95.0.4638.54 
- Safari Version 15.0 (16612.1.29.41.4, 16612)

<img width="889" alt="image" src="https://user-images.githubusercontent.com/8431042/139122910-088b2db9-fd1b-4f66-8f17-64bcb5a6bf03.png">

**Comment by Katsute** [comment, marked as answer]
You need to add at least one item before you can change the settings.

**Comment by maxcell** [comment]
Thank you! I definitely wouldn't have figured that out whatsoever 😅. I am glad that I had the feedback form because otherwise I just would have stopped.

**Reply by timoyates** [reply]
Thanks for the feedback @maxcell! Circling back with the team to see what we can do here. Cheers!
